docs/networking/cuda.rst

```diff
@@ -38,15 +38,30 @@ Open MPI offers two flavors of CUDA support:
       shell$ ./configure --prefix=/path/to/ucx-cuda-install --with-cuda=/usr/local/cuda --with-gdrcopy=/usr
 
       # Configure Open MPI this way
-      shell$ ./configure --with-cuda=/usr/local/cuda --with-ucx=/path/to/ucx-cuda-install <other configure params>
+      shell$ ./configure --with-cuda=/usr/local/cuda --with-cuda-libdir=/usr/local/cuda/lib64/stubs/ --with-ucx=/path/to/ucx-cuda-install <other configure params>
 
 #. Via internal Open MPI CUDA support
 
 Regardless of which flavor of CUDA support (or both) you plan to use,
 Open MPI should be configured using the ``--with-cuda=<path-to-cuda>``
-configure option to build CUDA support into Open MPI.
+and ``--with-cuda-libdir=<path-to-libcuda.so>`` configure options to
+build CUDA support into Open MPI.
 
-This affects the smcuda shared memory btl, as well as the uct btl.
+Open MPI supports building with cuda libraries and running on systems
+without cuda libraries or hardware. In order to take advantage of
+this functionality, when compiling, you have to specify the cuda
+dependent components to be built as dso's using the
+``--enable-mca-dso=<comma-delimited-list-of-cuda-components.``
+configure option.
+
+This affects the smcuda shared memory btl, rcache rgpusm, rcache
+gpusm, accelerator cuda, as well as the uct btl.
+
+An example configure command would look like the following:
+
+   .. code-block:: sh
+      # Configure Open MPI this way
+      shell$ ./configure --with-cuda=/usr/local/cuda --with-cuda-libdir=/usr/local/cuda/lib64/stubs --enable-mca-dso=btl-smcuda,rcache-rgpusm,rcache-gpusm,accelerator-cuda <other configure params>
```

Comment:
This is probably too wide.  Can you break it up into 2 lines, with a `\` continuation character?

Reply:
Done